rpc/src/rpc.rs

```diff
@@ -3856,22 +3951,22 @@ pub mod rpc_deprecated_v1_9 {
         fn get_recent_blockhash(
             &self,
             meta: Self::Metadata,
-            commitment: Option<CommitmentConfig>,
```

Comment:
While it may not matter a whole lot, I think it's weird to change the type signatures of all these deprecated endpoints.